00-introduction.md

```diff
@@ -185,6 +185,10 @@ See [BOLT #11: Invoice Protocol for Lightning Payments](11-payment-encoding.md)
    * _See category: node_
    * _See related: final node, processing node_
 
+* *Outpoint*:
```

Comment:
Not technically a BOLT 2 change, but it's used frequently enough that I realized it'd be nice to formally define. I recognize that it's pretty frequently used terminology within the bitcoin space, but it's nice to have it spelled out.